src/views/faq/faq.js

```diff
@@ -9,27 +9,17 @@ import { bitcoinLnd, litecoinLnd } from '../../constants';
 
 const twitterLink = 'https://twitter.com/boltzhq';
 
-const styles = theme => ({
+const styles = () => ({
   wrapper: {
+    flex: 1,
     alignItems: 'center',
-    justifyContent: 'space-around',
+    justifyContent: 'center',
+    overflowY: 'scroll',
   },
   tab: {
+    height: '100vh',
     width: '800px',
-    marginTop: '20px',
-    paddingTop: '20px',
-    paddingBottom: '20px',
-    paddingStart: '50px',
-    paddingEnd: '50px',
-    display: 'flex',
-    alignItems: 'center',
-    alignContent: 'center',
     flexDirection: 'column',
-    justifyContent: 'center',
-    backgroundColor: theme.colors.white,
```

Comment:
The background should be added again because right now the box is transparent

Reply:
Ill fix this in another PR for #95 I don't want to bloat this with too many unrelated changes.